portfolio/src/main/webapp/index.html

```diff
@@ -22,9 +22,20 @@ <h1>Adesemi Chloe Elumaro</h1>
       <p>Click here to see some of my favorite quotes:</p>
       <button onclick="addRandomGreeting()">Click for quotes :)</button>
       <div id="greeting-container"></div>
-      <p> Click for a greeting!<p>
-      <button onclick="getGreetingUsingAsyncAwait()">Click for a greeting :)</button>
-      <div id="greetings-container"></div>
+      
+      <form action="/data" method="POST">
+      
+      <p>Leave me a comment or let me know what you think of my portfolio!:</p>
+      <textarea name="text-input">Name,comment</textarea>
+      <br/><br/>
+      <input type="submit" />
+      </form>
+
+      <p> Click to see previous comments!<p>
+      <button onclick="getCommentUsingAsyncAwait()">Comments</button>
```

Comment:
Do you get comments by a button click. Seems like it makes a new request and returns an empty arraylist from the get call

Reply:
Clicking on the button shows the comments stored in myComments I believe